docs/docs/guides/migration_from_other_libs/ethers.md

```diff
@@ -0,0 +1,478 @@
+---
+sidebar_position: 1
+sidebar_label: 'Migration from ethers.js'
+title: 'Migration from ethers.js'
+---
+
+Follow this guide, if you're currently using the ethers.js library to interact with the Ethereum blockchain and want to migrate to web3.js.
+
+However, migrating from a library to another would usually need careful changes. But, ethers.js have lots of similarities with web3.js and migration would usually be easy and straightforward. However, you still need to check your code for possible tweaks as needed.
+
+## Installation
+
+First, install the latest version of web3.js:
+
+```bash
+npm install web3
+```
+
+## Initialization and getting the last block number
+
+With ethers.js, you would initialize like:
+
+```typescript
+import { ethers } from 'ethers';
+
+async function getBlockNumber() {
+  // in v5:
+  const provider = new ethers.providers.JsonRpcProvider(url);
+
+  // in v6:
+	const provider = new ethers.JsonRpcProvider(url);
+
+	const ts = provider.getBlockNumber();
+	ts.then(console.log);
+}
+// outputs something like: 18561956
+getBlockNumber();
+```
+
+With web3.js:
+
+```typescript
+import { Web3 } from 'web3';
+
+async function getBlockNumber() {
+	const web3 = new Web3(url);
+	const ts = web3.eth.getBlockNumber();
+	ts.then(console.log);
+}
+// outputs something like: 18561956n
+getBlockNumber();
+```
+
+## Use browser-injected provider
```

Comment:
I think following should be sequence:
1. providers section
2. wallet section
3. eth package section
4. contract package section
5. utils section
mostly sections are aligned with above sequence except this injected provider should be at start.